userguide/advanceduse/reverse-tethering.rst

```diff
@@ -15,60 +15,61 @@ Steps
 #. On the *device*:
 
    - Run: ``gdbus call --system --dest com.meego.usb_moded --object-path /com/meego/usb_moded --method com.meego.usb_moded.set_mode rndis_adb``
+   - Turn off Wifi and Data to avoid the gateways from clashing
+   - Bring your tethering connection down: Run ``nmcli connection down tethering``
+   - Modify your tethering connection: Run ``nmcli connection modify tethering ipv4.method auto``
 
 #. On your *computer*: 
 
-   - Get your RNDIS IP address:
-
-     - Run: ``ip a``
-     - Look for an interface named ``usb0`` in the output:
-
-       ::
-
-         5: usb0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UNKNOWN group default qlen 1000
-            link/ether 12:34:56:ab:cd:ef brd ff:ff:ff:ff:ff:ff
-            inet 10.42.0.149/24 brd 10.42.0.255 scope global noprefixroute usb0
-              valid_lft forever preferred_lft forever
-
-       - Your IP is: ``10.42.0.149``
-
-#. On the *device*:
+   - Get your interface name: Run ``ip route show to default via 10.42.0.1``
+     ::
 
-   - Turn off Wifi and Data to avoid the gateways from clashing
-   - Get your RNDIS interface: Run ``ip a``
+       default dev enx122626d0fe26 proto dhcp src 10.42.0.118 metric 101
 
+   - Your interface name is: ``enx122626d0fe26``
+   - Get your connection name: Run ``nmcli device show enx122626d0fe26``
+   - Look for a connection with the interface name you found in the output:
      ::
 
-       31: usb0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
-           link/ether 12:34:56:ab:cd:ef brd ff:ff:ff:ff:ff:ff
-           inet 10.42.0.1/8 brd 10.255.255.255 scope global rndis0
-             valid_lft forever preferred_lft forever
+       GENERAL.DEVICE:                         enx122626d0fe26
+       GENERAL.TYPE:                           ethernet
+       GENERAL.HWADDR:                         FA:F7:F5:D9:4C:9E
+       GENERAL.MTU:                            1500
+       GENERAL.STATE:                          100 (connected)
+       GENERAL.CONNECTION:                     Wired connection 2
+       GENERAL.CON-PATH:                       /org/freedesktop/NetworkManager/ActiveConnection/12
+       WIRED-PROPERTIES.CARRIER:               on
+       IP4.ADDRESS[1]:                         10.42.0.118/24
+       IP4.GATEWAY:                            10.42.0.1
+       IP4.ROUTE[1]:                           dst = 10.42.0.0/24, nh = 0.0.0.0, mt = 101
+       IP4.ROUTE[2]:                           dst = 0.0.0.0/0, nh = 10.42.0.1, mt = 101
+       IP4.DNS[1]:                             10.42.0.1
+       IP6.ADDRESS[1]:                         fe80::692:2a1:8127:91a1/64
+       IP6.GATEWAY:                            --
+       IP6.ROUTE[1]:                           dst = fe80::/64, nh = ::, mt = 1024
+
+   - Your connection name is ``Wired connection 2``
+   - Bring the connection down: Run ``nmcli connection down Wired\ connection\ 2``
```

Comment:
Consider quotes instead of backslashes to make it a little more human readble

nmcli connection down 'Wired connection 2'

Reply:
yes i agree, i'll do that :D